### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by frvanco**
Navbar OK, add animation OK

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ButtonWrapper = styled.section`
 
 const Button = styled.button`
   color: white;
-  border-radius: 20px;
+  border-radius: 20px;git
```
Comment (JulienDelemarle) [inline src/components/Header.jsx]: delete "git"

**Review by JulienDelemarle — APPROVED** [review]
ok

**Review by JulienDelemarle — DISMISSED** [review]
one small error to delete: "git" line14

**Review by JulienDelemarle — DISMISSED** [review]
conflicts resolved

File: src/App.jsx [hunk 2]
```diff
@@ -22,6 +22,9 @@ function App() {
           <Route path="/quiz">
             <QuizPage />
           </Route>
+          <Route path="/results/:int/:spe/:str/:pow">
+            <p>?</p>
+          </Route>
```
Comment (LoicBrassart) [inline src/App.jsx]: Haha, no, add a real route here please ;-)

File: src/components/Header.jsx [hunk 3]
```diff
@@ -29,6 +30,28 @@ const Button = styled.button`
...
+    position: absolute;
+    background: #fff;
+    transition: width 0.3s ease 0s, left 0.3s ease 0s;
+    width: 0;
+  }
+  &:hover:after {
+    width: 100%;
+    left: 0;
+=======
+>>>>>>> d2a47a918240d52a38bf9e15736dd2e9b98cb2cc
```
Comment (LoicBrassart) [inline src/components/Header.jsx]: Oh-oh, there are conflicts artifacts here ! :scream:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils to fix

**Review by JulienDelemarle — APPROVED** [review]
fixed problems

**Review by LoicBrassart — APPROVED** [review]
No More Evil !

**Pull request merged**